internal/command/instance_idp.go

```diff
@@ -560,6 +553,29 @@ func (c *Commands) UpdateInstanceSAMLProvider(ctx context.Context, id string, pr
 	return pushedEventsToObjectDetails(pushedEvents), nil
 }
 
+func (c *Commands) GenerateInstanceSAMLProvider(ctx context.Context, id string) (*domain.ObjectDetails, error) {
```

Comment:
where is that used?

Reply:
looking at the prepare function i think it's for regenerating the key / cert, i guess there should be an endpoint in Admin and Mgmt.
And i'd then name it something like `RegenerateInstanceSAMLProviderCertificate`